benchopt/base.py

```diff
@@ -259,7 +259,7 @@ def warm_up(self):
         ...
 
     def _warm_up(self):
-        if not getattr(self, '_warmup_done', True):
+        if not getattr(self, '_warmup_done', False):
```

Comment:
It seems to me that this implementation is a bit unclear

WDYT about this suggestion @MatDag?

```suggestion
        
        if getattr(self, '_warmup_done', None):
            # already warmed up
            return
        
        self.warm_up()
        self._warmup_done = True
```

Reply:
Seems clearer indeed